### ISSUE TITLE
undefined reference to `_Unwind_Resume'

### ISSUE THREAD

**Issue by mwarning** [msharov/ustl#4]
ok, this is probably not a ustl bug...
Compiling the vector example from the docs gives me the folloeing undefined reference:
```
g++ -nodefaultlibs -lc test.cpp -L/usr/local/lib -lustl -o test
/tmp/ccUYAka8.o: In function `main':
test.cpp:(.text+0x100): undefined reference to `_Unwind_Resume'
collect2: error: ld returned 1 exit status
```
Any idea what could be wrong?

**Comment by msharov** [comment]
_Unwind_Resume is defined in libgcc_s.so, and you need to add -lgcc_s to the link command. This should have been mentioned in the documentation, which currently lists "-nodefaultlibs -lc" as the command for not linking to libstdc++ - a problem now fixed by commit 6414ce287d2e3800fbfca7632a264ff7c309f2d1

The easier way to do this is to link using gcc instead of g++, in which case libstdc++ is always omitted by default. This *is* mentioned in the documentation.

**Comment by msharov** [comment]
Why do people still keep coming here a dozen times a week? I thought I had fixed all the documentation to explain the correct linking method (with gcc). If you are having this problem, please leave a suggestion here on how the documentation could be changed to direct you to the correct solution sooner.
